docs/npwd/Credits.md

```diff
@@ -4,21 +4,22 @@ title: Contributers
 sidebar_label: Contributers
 ---
 
-We appreciate all the support and help we get. Looking forward to making this list longer!
+We appreciate all the support and help we get. Below is a list of contributors to the resource with the [**Project Error Team**](https://github.com/orgs/project-error/people) removed from the list. We hope we continuously add to this list as time passes.
```

Comment:
Would probably be better worded with "Below is a list of community contributors"

Reply:
Will focus test this with the focus group.